src/main/java/innovimax/mixthem/operation/CellOperation.java

```diff
@@ -0,0 +1,24 @@
+package innovimax.mixthem.operation;
+
+import innovimax.mixthem.arguments.ParamValue;
+
+public enum CellOperation { 
+  DEFAULT_SPLIT_CELL_REGEX("\\s"),
+  DEFAULT_CELL_SEPARATOR(" ");
+
+  private ParamValue value;
```

Comment:
why paramValue ? why not just String ?

Reply:
Used as default value for parameters in Map.getOrDefault().
Another way is to create default value when used with ParamValue.createString().